extensions/schema-compare/src/schemaCompareMainWindow.ts

```diff
@@ -23,6 +23,8 @@ const generateScriptNoChangesMessage = localize('schemaCompare.generateScriptNoC
 const applyEnabledMessage = localize('schemaCompare.applyButtonEnabledTitle', "Apply changes to target");
 const applyNoChangesMessage = localize('schemaCompare.applyNoChanges', "No changes to apply");
 const includeExcludeInfoMessage = localize('schemaCompare.includeExcludeInfoMessage', "Please note that include/exclude operations can take a moment to calculate affected dependencies");
+const SourceTitle: string = localize('schemaCompareDialog.SourceTitle', "Source");
```

Comment:
Var names should be camel cased (e.g. sourceTitle) and the string qualifier here is redundant so I'd just remove it (we already know it's a string because it's just a localize call

Reply:
Fixed.